downloads/index.html

```diff
@@ -52,10 +52,10 @@ <h1 style="font-size: 0.0em">Lawnchair Downloads</h1>
           <source type="image/webp" srcset="/images/lawnchair.webp" >
           <img src="/images/lawnchair.png" width="96" height="96" alt="The Lawnchair Logo" >
         </picture>
-        <h2 class="card__name" title="The actual launcher.">Lawnchair</h2>
+        <h2 class="card__name" id="lawnchair_title" title="The actual launcher.">Lawnchair</h2>
         <span class="card__version" id="js-lawnchair-version">Loading...</span>
       </div>
-      <div class="card__right">
+      <div class="card__right" id="card_main_cta">
```

Comment:
You can remove the class names and use selectors instead (See my comment in `assets/downloads.css`).